code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -441,6 +441,14 @@
 	flags_inv = HIDEGLOVES|HIDESHOES|HIDEJUMPSUIT
 	allowed = list(/obj/item/clothing/mask/facehugger/toy)
 
+/obj/item/clothing/suit/nemes

```

Comment:
I keep reading this as "memes" please fix

Reply:
I can't fix stupid